src/commands/cmd_json.cc

```diff
@@ -291,6 +291,64 @@ class CommandJsonArrPop : public Commander {
   int64_t index_ = -1;
 };
 
+class CommanderJsonArrIndex : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    if (args.size() < 4 || args.size() > 6) {
```

Comment:
```suggestion
    if (args.size() > 6) {
```

Line 361 required the number of arguments must be at least 4, so can remove this check.

Reply:
fixed it， please check